compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -1192,6 +1192,9 @@ struct RootCollector<'a, 'tcx> {
 
 impl<'v> RootCollector<'_, 'v> {
     fn process_item(&mut self, id: hir::ItemId) {
+        if self.tcx.hir().maybe_body_owned_by(id.def_id.def_id).is_some() {
+            self.tcx.ensure().mir_borrowck(id.def_id.def_id);
```

Comment:
We clearly need a `-Zfuckit-completely` that removes this line as well.

Reply:
-Zi-am-normal-and-can-be-trusted-with-mir-borrowck

Reply:
-Zi-borrowck-myself-dont-worry

Reply:
-Zthe-regions-and-i-are-one